src/mlte/measurement/process_measurement.py

```diff
@@ -0,0 +1,94 @@
+"""
+Base class for measurement of external processes asynchronally.
+"""
+
```

Comment:
Can we add unit tests (standalone, and within the measurements that subclass `ProcessMeasurement`) to verify this functionality?

Reply:
We don't have any code-coverage rules in place so far, but this is something I would like to add in the future, so keeping up with it now reduces the burden.

Reply:
Ok, will do